integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/End2EndTestITStep.java

```diff
@@ -56,18 +58,18 @@ public void authenticationIsEnabled(int responseCode) {
                 .statusCode(responseCode);
     }
 
-    @Given("^get list of Bridge instances with access token doesn't contain Bridge \"([^\"]*)\"$")
-    public void getEmptyBridges(String bridgeName) {
+    @Given("bridge with randomly generated name doesn't exist")
```

Comment:
What about leaving the old step text there, as it contains context info about containing access token?